bin/common.bash

```diff
@@ -105,7 +105,7 @@ check_tools() {
   warn=0
   err=0
 
-  for executable in jq yq4 s3cmd sops kubectl helm helmfile dig pwgen htpasswd; do
+  for executable in jq yq4 s3cmd sops kubectl helm helmfile dig pwgen htpasswd az; do
```

Comment:
I don't think `az` is a requirement here, only the script, right?

Reply:
The script needs az, so i thought it should be set a requirement there.

Reply:
I _think_ this is only a check for tools that are required for running things under `bin/`. Please correct me if I'm wrong though!

Reply:
So, My instinct directly led me to check s3cmd, which is technically invoked through `bin/` :laughing: , but i believe that we will be do the same with the azure creation, or maybe remove the s3cm command and make it run the proper thing depending on the storage type.

But, i think we can remove it for now until we decide on the implementation there.

Reply:
I'd say that we should not introduce this by the same rationale as why #2060 dropped the introduction of `CK8S_CLUSTER_PROVISIONER`.

The integration to make Azure work is targeting beta, and by introducing this we introduce an Azure CLI dependency for _everyone_ using Apps, and we should resolve the part from #2000 to make such dependencies optional depending on the infrastructure provider.

Reply:
Will remove the azure-cli parts before merging then.